modules/nf-core/glimpse/ligate/main.nf

```diff
@@ -8,7 +8,7 @@ process GLIMPSE_LIGATE {
         'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
 
     input:
-    tuple val(meta), path(input)
+    tuple val(meta), val(input_list)
```

Comment:
```suggestion
    tuple val(meta), path(input_list)
```

This way the files get staged correctly (it's better when running with data stored on the cloud)